typhon/plots/common.py

```diff
@@ -321,3 +325,42 @@ def key(line):
                           ax.get_legend_handles_labels()[0]],
                          *ax.get_legend_handles_labels()),
                      reverse=reverse)))
+
+
+def write_multi(fig, name, exts=["png", "pdf", "pkz"]):
+    """Write figure to multiple files
+
+    Writes a matplotlib Figure object to multiple files.  By default, it
+    writes:
+
+        * png
+        * pdf
+        * pkz, pickle containing the figure object
+
+    but this can be configured by passing the ``exts`` argument.  Most will
+    be passed directly to ``print_figure``, except ``pkz``, which will use
+    ``pickle.dump``.
+
+    Still planned are special cases for ``info``, which will dump stack
+    information, and "tikz", which will use matplotlib2tikz to write a file
+    for use with the LaTeX pgfplots library.
```

Comment:
Since this is more an internal developer note, it should not be in the user documentation. Either remove it or move it from the docstring to a comment.